tests/models/test_cohort.py

```diff
@@ -0,0 +1,585 @@
+import json
```

Comment:
Consider reorganizing the test per the style: https://github.com/NCI-GDC/gdc-dev-guide/wiki/style#unit-test-layout. @BenX00 would the top level 'gdc_ng_models' also be part of the path like 'tests/unit/gdc_nd_models/models/cohort.py ?

Reply:
Thank you Jason! No time like the present to start implementing our checklist. I've reorganized the tests (without including gdc_ng_models in the top level). If we want to open this up to wider discussion, I'm happy to revisit and make additional changes.

Reply:
The Use Case for including the root package name of gdc_nd_models is if we ever started merging projects into a single repo that produces multiple output products which had been discussed at one point. I think that is a bigger can of worms than I'm willing to open right now. Proceed.